snooty.toml

```diff
@@ -18,6 +18,8 @@ toc_landing_pages = [
   "/migrate-to-atlas-cli"
   ]
 
+eol = true
```

Comment:
I would put this on line 3 because it says it should be at the top of the file. I'm not POSITIVE that this won't work as-is, but I know the placement matters so just to be safe, that's where I've always put it